wp-content/themes/pub/wporg-learn-2020/archive-lesson-plan.php

```diff
@@ -113,7 +109,11 @@
 					) );
 
 					foreach ( $levels as $level ) : ?>
-					<li><a href="<?php echo esc_url( get_post_type_archive_link( 'lesson-plan' ) ) . '?level[]=' . esc_html( $level->term_id ); ?>"><?php echo esc_html( $level->name ); ?><span class="dashicons dashicons-arrow-right-alt2"></span></a></li>
+					<li>
+						<a class="button" href="<?php echo esc_url( get_post_type_archive_link( 'lesson-plan' ) ) . '?level[]=' . esc_html( $level->term_id ); ?>">
+							<?php echo esc_html( $level->name ); ?><span class="dashicons dashicons-arrow-right-alt2"></span>
```

Comment:
I think these dashicons will need `aria-hidden` too? They aren't read aloud by VoiceOver but since you were given that feedback for the topic/audience ones, they could be by another screen reader. Maybe add `aria-hidden` to all the places dashicons are used.

```suggestion
							<?php echo esc_html( $level->name ); ?><span aria-hidden="true" class="dashicons dashicons-arrow-right-alt2"></span>
```

Reply:
Done, thanks